index.bs

```diff
@@ -1035,7 +1052,7 @@ In these situations, as with the typical flow, the player keeps iframe invisible
 		<li>Player posts `Player:init` message after ad media playback started.</li>
 		<li>Creative proccesses initialization data and finalizes assets loading. Sub-loading routines may cause latencies.</li>
 		<li>Creative responds with [[#simid-player-init-resolve]].</li>
-		<li>Player posts [[#simid-player-startCreative]] when it is ready to render interactive component.</li>
+		<li>Player posts [[#simid-player-startCreative]] immediately.</li>
 	</ol>
 </div>
```

Comment:
right underneath this section is a bunch of definitions of message args.  This is definitely out of order and needs to be fixed.